data/human/culture conversations.txt

```diff
@@ -1101,3 +1101,279 @@ mission "Little Vienna in New Austria"
+	word
+		"at the starting line"
+		"in front of the starting grid"
+	word
+		", and the hovercraft launch forward in a burst of speed."
+		", then engines ignite in a frenzy to send the sleek machines darting into the first lap."
+		", as the racers enter into a chaotic scramble for the lead."
+		" - hovercraft fighting for position as one leaps ahead of the rest."
+	word
+		" "
+	word
+		"An hour in, the racers continue to weave through the course's twisting tracks, "
+		"Midway, the competition transforms into a dynamic dance, "
+		"The middle phase entails a symphony of sounds, "
+		"As the race unfolds, a few hovercraft nearly crash before recovering, "
+	word
+		"each maneuver a blend of precision and raw power."
+		"with daring overtakes and strategic maneuvers."
+		"each moment getting closer to the crescendo of the race's final hour."
+		"as competitors push their vehicles to the limit at every turn."
+
+phrase "sunracer hovercraft victory description"
+	word
+		" surges ahead, their craft a blur of power and precision, leaving rivals in their wake by multiple laps to finish the race in a clear victory."
+		" clinches victory in a breathtaking display, a decisive lap ahead of the field."
+		" edges out the competition in a photo-finish, securing the win by a razor-thin margin."
+		" rockets to an unrivaled lead in the race's last few minutes, ending the race with a clear, commanding advantage."
+	word
+		" "
+	word
+		"The middle two racers converged into a frantic dash for second while one lone racer lagged far behind."
+		"The other three teams bundled together in a last-minute sprint to complete the race."
+		"The rest clustered tightly for the race finish."
+		"The remaining racers finished uneventfully spread out."
+		"Only three racers finished today, as one crashed off the track, luckily with the driver unharmed."
+	word
+		" "
+	word
+		"With the race finished, the crowd gradually disperses, and you wander back to your <model> amid the soft buzz of post-race chatter."
+		"As the stands empty, you slip away through clusters of lingering fans back to the <ship>."
+		"Despite the race being over, the crowd still roars with adrenaline, and you still feel the crowd's excitement as you return to your ship."
+		"The venue quiets as spectators scatter, leaving you to drift back to your <model> in the calm that's followed the thrill of the race."
+
+mission "A Sunracer Hovercraft Racing"
+	minor
+	repeat
+	invisible
+	to offer
+		random < 40
+	source "Sunracer"
+	substitutions
+		"<racename>" "${adjectives} ${racetrack}"
+		"<megaparsec name>" "${civilian}"
+		"<syndicate name>" "${syndicate fighter}"
+		"<delta name>" "${free worlds fighter}"
+		"<deep name>" "${deep fighter}"
+	on offer
+		conversation
+			`${sunracer opening}, a ${sunracer hovercraft adhook}: there's a hovercraft race at the <racename>, just a few hours away by hover rail, or much faster aboard a <model>. ${sunracer hovercraft adcloser}.`
```

Comment:
```suggestion
			`${sunracer opening}, a ${sunracer hovercraft adhook}: there's a hovercraft race at the <racename>, just a few hours away by hover rail, or much faster aboard a <model>. ${sunracer hovercraft adcloser}`
```

See above. I'd recommend moving the period into the "sunracer hovercraft adcloser" phrase.